common/match/pool.ts

```diff
@@ -386,9 +386,6 @@ export async function runMatching(poolName: string, apply: boolean, _toggles: st
     if (apply) {
         const startCommit = Date.now();
         for (const match of matches) {
-            if (pool.confirmInterest && match.pupil.openMatchRequestCount <= 1) {
-                await removeInterest(match.pupil);
-            }
```

Comment:
This can stay

Reply:
Is there a case where it's needed?

Reply:
Ah nevermind, we call createMatch right after which now calls it anyways